tests/lib/rules/no-return-await.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @fileoverview Disallows unnecessary `return await`
+ * @author Jordan Harband
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-return-await");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const parserOptions = { ecmaVersion: 2017 };
```

Comment:
Nit: I think you can pass this option as a default to the `RuleTester` constructor, to avoid repeating it for every test.

```js
const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2017 } });
```